<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plantillaExamen</class>
 <widget class="QWidget" name="plantillaExamen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>391</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Informacion Paciente</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>60</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Apellidos:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_apellidos">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>20</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombres:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_nombres">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>213</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Fecha Nacimiento:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Remitido por:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_doctor"/>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_2</zorder>
   <zorder>line_apellidos</zorder>
   <zorder>label</zorder>
   <zorder>line_nombres</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>531</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Hematologia</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Siguiente</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Atras</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
